docs/source/tutorial_chat_service.md

```diff
@@ -44,6 +46,54 @@ additional_args:
   page_id: 1 # configure your own page
 ```
 
+Let's walk through each of these keys:
+
+- **`tasks`** - the `tasks` key defines the options that are presented to a user upon entering an `Overworld`. In this case, there is only one `task` defined: `default`. The `default` key specifies the following subkeys:
+    - `onboard_world` - the world used for agent onboarding
+    - `task_world` - the world that defines that logic for handling users in the main task
+    - `timeout` - the timeout for human responses in the world
+    - `agents_required` - how many *human* agents are required to run this world. Specify > 1 if you'd like more than one human to interact.
+- **`task_name`** - the name of your task. not super important to define, but good to have nonetheless
+- **`world_module`** - this is a module path to the module in which **all** of the task worlds are defined. If you have worlds in separate files, simply importing them into this file will suffice.
+- **`overworld`** - The name of the overworld class.
+- **`max_workers`** - How many max human connections you plan on supporting for your service
+- **`opt`** - any task-specific options you need to define.
+    - **`models`** - Perhaps **the most important** key in this config. This is where you define the models that you'll be using. Each entry under this is a mapping from custom model name to model params. So, in this example, we have defined a model `blender_90M` that we would like to access in their task. For this model, we then specify the actual model arguments necessary to create a ParlAI agent.
+- **`additional_args`** - These are for any specific service-specific args you might want to specify - e.g. if using Facebook Messenger, you would specify your page ID here.
+
+:::{important}
+**ON THE SUBJECT OF MODELS**
```

Comment:
If you put this on the previous line, it will be the "header" of the note

Reply:
Just tried that and I think I prefer this way